### PR TITLE
Replace flash service with angular-toastr

### PR DESCRIPTION
**Pull request by tilgovi**
1. The original code was written to replace `Annotator.Notification` but Annotator is no longer used in the app frame. Only the auth code still uses the old notifications, and is on the chopping block to be replaced within the week.
2. This code relies on jQuery and I found it to have the most jQuery-specific bits of all the code I audited. In particular, the show/hide animation via slideUp and slideDown, would be implemented in Angular in a _completely_ different way.
3. This works and it's attractive and it supports icons and HTML and other customizations.

### CODE REVIEW

**Comment by tilgovi** [comment]
Adds about 5kB to our payload. On the other hand, I've got a branch already where I've replaced the rest of our jQuery-isms so that when we drop Annotator Auth Plugin we shed 150kB. I propose we revisit this if we think we need to shed weight later, but this is a win overall for now.

**Comment by nickstenning** [comment]
Very happy with replacing our `flash` service with some code that we don't need to maintain, but I think it would be better to have a layer of abstraction between our code and toastr. I'd suggest being the flash service back as a dumb wrapper around toastr, so that the only references to toastr in our code are in one place. That way if we do revisit this decision we only have to change code in one place.

**Comment by tilgovi** [comment]
Truth

**Comment by tilgovi** [comment]
Ready to go.

File: h/static/scripts/account/account-controller.coffee [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class AccountController
   @inject = [  '$scope', '$filter',
-               'auth', 'flash', 'formHelpers', 'identity', 'session']
+               'auth', 'formHelpers', 'identity', 'flash', 'session']
```
Comment (nickstenning) [inline h/static/scripts/account/account-controller.coffee]: A tiny niggle, but let's avoid no-op changes like this in the history, if possible.

File: h/static/scripts/account/account-controller.coffee [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   constructor: ($scope,   $filter,
-                auth,   flash,   formHelpers,   identity,   session) ->
+                auth,   formHelpers,   identity,   flash,   session ) ->
```
Comment (nickstenning) [inline h/static/scripts/account/account-controller.coffee]: Ditto.

**Comment by nickstenning** [comment]
LGTM modulo one tiny dealphabetisation error. :)

**Comment by tilgovi** [comment]
Shoot. I thought I went back and fixed those. On it.

**Comment by tilgovi** [comment]
Done.

**Pull request merged**